src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -317,6 +324,11 @@ const reportActionsSelector = (reportActions) =>
     reportActions &&
     _.map(reportActions, (reportAction) => ({
         errors: reportAction.errors,
+        message: [
+            {
+                moderationDecisions: [{decision: lodashGet(reportAction, 'message[0].moderationDecisions[0].decision')}],
+            },
+        ],
```

Comment:
Added this so SidebarLinks rerender when the `moderationDecisions` value changed